spec/lib/hackney/income/migrate_uh_agreement_spec.rb

```diff
@@ -35,8 +33,8 @@
     it 'does not call anything' do
       expect(view_agreements).not_to receive(:execute)
       expect(view_court_cases).not_to receive(:execute)
-      expect(create_informal_agreement).not_to receive(:execute)
-      expect(create_formal_agreement).not_to receive(:execute)
+      expect(create_agreement).not_to receive(:execute)
+      expect(create_agreement).not_to receive(:execute)
```

Comment:
```suggestion
      expect(create_agreement).not_to receive(:execute)
```
slightly nitpicky but also i don't know rspec too well, do we need both of these here? 🤔

Reply:
I assume not, but line 37 is redundant

Reply:
```RUBY
CreateInformalAgreement < CreateAgreement
```
Yep CreateInformalAgreement/CreateFormalAgreement is a `CreateAgreement`